content/en/docs/ibm/install-kubeflow.md

```diff
@@ -31,67 +31,54 @@ deploy Kubeflow on IBM Cloud.
 To simplify the command lines for this walkthrough, you need to define a few
 environment variables.
 
-1.  Set `CLUSTER_NAME`, `CLUSTER_REGION`, and `CLUSTER_ZONE` variables:
+1.  Set `CLUSTER_NAME` and `CLUSTER_ZONE` variables:
 
     ```
     export CLUSTER_NAME=kubeflow
-    export CLUSTER_REGION=us-south
     export CLUSTER_ZONE=dal13
     ```
 
     - `CLUSTER_NAME` must be lowercase and unique among any other Kubernetes
-      clusters in this IBM Cloud region.
-    - `CLUSTER_REGION` can be any region where IKS is available. You can get a
-      list of all available regions via the
-      [IBM Cloud documentation](https://console.bluemix.net/docs/containers/cs_regions.html#regions-and-zones)
-      or via `ibmcloud cs regions`.
-    - `CLUSTER_ZONE` can be any zone that is available in the specified region
-      above. You can get a list of all available locations from the
-      [IBM Cloud documentation](https://console.bluemix.net/docs/containers/cs_regions.html#zones)
-      or by using `ibmcloud cs zones` after you set the region by using
-      `ibmcloud cs region-set $CLUSTER_REGION`.
+      clusters in the specified CLUSTER_ZONE.
+    - `CLUSTER_ZONE` identifies the location where CLUSTER_NAME will be created. Run `ibmcloud ks locations` to list supported IBM Cloud Kubernetes Service locations. For example, choose `dal13` to create CLUSTER_NAME in the Dallas (US) data center. 
 
 ### Creating a IBM Cloud Kubernetes cluster
 
 To make sure the cluster is large enough to host all the Knative and Istio
 components, the recommended configuration for a cluster is:
 
-- Kubernetes version 1.15 
+- Kubernetes version 1.16 (or later)
```

Comment:
Question: do we really need Kubernetes 1.16 (or later)?
According to https://www.kubeflow.org/docs/started/k8s/overview/#minimum-system-requirements,
Kubeflow 1.0 is incompatible with Kubernetes 1.16

As I recall, there is a recent PR to change this to 1.15 in IBM page.

Reply:
My apologies. I incorrectly assumed that Kubeflow has already been successfully validated on version 1.16. I will revert the change. I believe there would be value in including the system requirements link you've provided, especially because the default version on IBM Cloud is 1.16. The recommended version (1.14 according to the reqs doc) is no longer supported.

Reply:
I just realized that because the default Kubernetes version is not currently supported we will have to augment the `cluster create` commands with a parameter that chooses a compatible version. The caveat is though that over time the supported versions will change and the doc would require periodic updates.

Reply:
I reverted the change back to 1.15. Let me know what your preference is with respect to adding a link to the requirements and updating the CLI command examples to require version 1.15.

Reply:
1.16 should be supported now but maybe Kubeflow haven't claim it officially. Probably is safer to keep it in 1.15 until Kubeflow make the official announcement.